revdb/src/sqlite_connection.cpp

```diff
@@ -439,7 +439,7 @@ bool queryCallback(void *p_context, int p_placeholder, DBBuffer& p_output)
 		else
 		{
 			// According to documentation in sqlitedecode.cpp, this is the required size of output buffer
-			t_escaped_string = malloc(2 + (257 * t_parameter_value . length) / 254);
+			t_escaped_string = malloc(2 + (257 * (long)t_parameter_value . length) / 254);
```

Comment:
You need to use int64_t here and not 'long' as the size of long depends on processor bitness and arch (32-bit non-Windows - long is 32-bit, 64-bit non-Windows - long is 64-bit; 32-bit Windows - long is 32-bit, 64-bit Windows - long is 32-bit).